react-events/my-app/src/CustomButton.jsx

```diff
@@ -0,0 +1,20 @@
+import './CustomButton.css';
+
+// export default function CustomButton({ text, color }) {
+//   function handleClick() {
+//     window.alert(text);
+//   }
+//   return (
+//     <button onClick={handleClick} className={color}>
+//       {text}
+//     </button>
+//   );
+// }
+
+export default function CustomButton({ text, color, onCustomClick }) {
+  return (
+    <button onClick={onCustomClick} className={color}>
```

Comment:
This is where you would add the callback function to your `onClick` and then call `onCustomClick` inside of there with your `text` & `color` props